auditbeat/module/file_integrity/fileinfo_posix.go

```diff
@@ -67,6 +68,11 @@ func NewMetadata(path string, info os.FileInfo) (*Metadata, error) {
 		fileInfo.Owner = owner.Username
 	}
 
+	getExtendedAttributes(path, map[string]*string{
```

Comment:
Assuming this is linux-only then can we guard this with a `if runtime.GOOS == "linux"`.

Reply:
The `system.posix_acl_access` should in principle be available on other OSs.